R/data-combined.R

```diff
@@ -471,14 +484,20 @@ DataCombined <- R6::R6Class(
         return(data)
       }
 
-      # Note that `name` column is based on lexical order of names.
-      #
-      # This always works because the order of data frame (and thus the `name`
-      # column) returned by `*ToDataFrame()` functions is never changed.
-      data %>%
-        tidyr::nest(data = -name) %>%
-        dplyr::mutate(name = names) %>%
-        tidyr::unnest(cols = c(data))
+      # Create a nested data frame where all columns except `name` are collapsed
+      # into a single column named `data`, which will contain these columns in a
+      # list data frame.
+      data <- tidyr::nest(data, data = -name)
```

Comment:
@IndrajeetPatil Did not you mentioned that using pipe was also more efficient for memory usage? or am I imaginating it?